src/Filters/SessionAuth.php

```diff
@@ -49,6 +49,15 @@ public function before(RequestInterface $request, $arguments = null)
                 $authenticator->recordActiveDate();
             }
 
+            // Block inactive users when Email Activation is enabled
+            $user = $authenticator->getUser();
+            if ($user !== null && ! $user->isActivated()) {
+                $authenticator->logout();
+
+                return redirect()->route('login')
+                    ->with('error', lang('Auth.activationBlocked'));
```

Comment:
This message is suitable if the user has not activated it himself.
![error](https://user-images.githubusercontent.com/9530214/210578238-cf26b0db-d53d-4e76-8341-4268188804a3.jpg)

Suppose the user is activated. For some reason, the administrator has disabled it.

Is it appropriate to display the following message now?

Reply:
We do not have any way to know whether the admin did it or the user has never activated currently. I think that's likely ok, though. For applications where an admin might remove it, they could easily change the language string to reflect a more generic situation. 

Banning might be more appropriate in these situations, also?

Reply:
I agree with @lonnieezell 

If an admin deactivates a user, the error message seems to be still appropriate.
Only thing the user to do is to activate again.
If the message is not appropriate or ambiguous, devs would change the message.

Activation is not banning. I don't see use cases that an admin deactivates users.